README.md

```diff
@@ -1,104 +1,56 @@
-aims to bring Government departments online in a consistent and
-user-friendly manner. Documentation can be found on [rdoc](http://rdoc.info/github/alphagov/whitehall/frames).
+whitehall is a Ruby on Rails content management application for content published by government departments and agencies.
 
 ## Nomenclature
 
-* *Govspeak* A variation of [Markdown](https://daringfireball.net/projects/markdown)
-used throughout 'Whitehall' as the general publishing format
+- *Govspeak* A variation of [Markdown](https://daringfireball.net/projects/markdown) used throughout whitehall as the general publishing format
 
-## Technical Documentation
+## Technical documentation
 
-Whitehall is a Rails 5 app built on a MySQL database. It is deployed
-in two modes, 'admin' for publishers to create and manage content and
-'frontend' for rendering content under https://www.gov.uk/government. In addition to
-storing and managing its own content database Whitehall also updates
-various other APIs including search and is currently being migrated
-towards a new publishing model utilising [Publishing
-API](https://github.com/alphagov/publishing-api) and
-[Content Store](https://github.com/alphagov/content-store).
+whitehall is a Ruby on Rails app built on a MySQL database. It is deployed in two modes: 'admin' for publishers to create and manage content and 'frontend' for rendering some content under https://www.gov.uk/government and https://www.gov.uk/world. whitehall also sends most content to the Publishing Platform and search.
 
-## Dependencies
-
-* Xcode (for the Command Line Tools `xcode-select --install`)
-* Ruby 2.2.3
-* Rubygems and Bundler
-* MySQL
-* Imagemagick and Ghostscript (for generating thumbnails of uploaded
-  PDFs)
-* xpdf (first download [XQuartz](http://www.xquartz.org/))
-* PhantomJS (for running the Javascript tests)
-
-## Dependencies on Whitehall APIs
+Assets are currently handled by whitehall. It handles uploads, virus scanning and serving of assets. There is a current project to migrate whitehall to use [Asset Manager](http://github.com/alphagov/asset-manager).
 
-The following applications have dependencies on Whitehall APIs:
+## Dependencies
 
-* [alphagov/smart-answers](https://github.com/alphagov/smart-answers) uses the Worldwide
-  Locations and Worldwide Organisations endpoints
+- [alphagov/asset-manager](http://github.com/alphagov/asset-manager): provides uploading for static files (migration in progress)
+- [alphagov/rummager](http://github.com/alphagov/rummager): allows documents to be indexed for searching in both Finders and site search
+- [alphagov/publishing-api](http://github.com/alphagov/publishing-api): documents are sent here, persisted and then requested.
 
 ## Running the application
 
-The application can be started with
-
 ```
-bundle exec rails s
+$ ./startup.sh
 ```
+If you are using the GDS development virtual machine then the application will be available on the host at http://whitehall-admin.dev.gov.uk/
```

Comment:
Might be worth linking to the docs for the dev vm here?

Reply:
Yup, done